Sources/Quick/Behavior.swift

```diff
@@ -4,7 +4,11 @@
 
 open class Behavior<Context> {
 
-    public static var name: String { return String(describing: self) }
+    /**
+     Override this variable if you want to provide custom name for this example group.
+    */
+    open class var name: String { return String(describing: self) }
```

Comment:
This is exactly what I thought before and is not a breaking change, so it's safe to merge.

https://github.com/Quick/Quick/pull/821#discussion_r220907011